foladipo/docs-cabinet-cp2
# Feature/148219125/enable admin to update user accounts

#### What does this PR do?

This PR enables admin users to update the profile of other users, (including admin users). It also enables them to delete any user's account.

#### Description of task to be completed

Enable admin users to update user profiles or delete user accounts.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have set up an admin account. Log in to the app using the credentials you set.

- In the app menu, you will see an option to tagged 'Users'. Click on it to see a list of all users.

- Click on the edit icon (colored green) of one of the users.

- You can now go ahead to update that user's profile. You can delete his/her account too.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148219125

#### Screenshots
<img width="1269" alt="screen shot 2017-07-05 at 4 22 15 pm" src="https://user-images.githubusercontent.com/26963575/27871643-2b1995ec-619e-11e7-8da0-fa9984b80c5c.png">
